doc/developer/documentationAraInstall.md

```diff
@@ -0,0 +1,78 @@
+
+ ## Installing ARA for development
+
+ ### Install the required tools
+ 
+ + Install the following:
+   + Java 8
+   + Maven
+   + IntelliJ Idea (_back development_)
+   + Visual Studio Code or IntelliJ / WebStorm Idea Ultimate Edition (_front development_)
+   + Docker
+   + Git
+
+ + Then verify that they are successfully installed:
+   + Java `$ java --version` (the version should be Java 8)
+   + Maven `$ mvn --version` (the displayed Java version should be Java 8)
+   + Docker `$ docker --version`<br>
+   + Git `$ git --version` <br>
+   
+ ![Check versions](imagesInstallARA/version_control.PNG)

```

Comment:
Could you change the images link to prefix them with `./` ? In order to make a real link for GitHub preview of AsciiDoc ?

Reply:
(Apply for all links)